tests/Validator/Constraint/VatNumberValidatorTest.php

```diff
@@ -61,30 +63,50 @@ public function testValidateWithEmptyValueWillReturnWithNoViolation()
         $this->api->expects(self::never())->method('validateVat');
         $this->context->expects(self::never())->method('addViolation');
 
-        self::assertNull($this->validator->validate(null, $this->constraint));
+        $this->validator->validate(null, $this->constraint);
     }
 
-    public function testValidateWithInValidVatNumberWillReturnWithViolation()
+    public function testValidateWithInValidConstraintWillReturnWithViolation()
```

Comment:
This test method name seems misleading, there is asserted *no* validation is added